sdk/identity/identity/review/identity.api.md

```diff
@@ -56,6 +56,18 @@ export interface AuthenticationRequiredErrorOptions {
     scopes: string[];
 }
 
+// @public
+export interface AuthorityValidationOptions {
+    // (undocumented)
+    authorityMetadata?: string;
+    // (undocumented)
+    cloudDiscoveryMetadata?: string;
```

Comment:
These 3 other options (authorityMetadata, cloudDiscoveryMetadata, skipAuthorityMetadataCache) are not universally supported by other MSALs and there are no plans to do so.